src/components/index.tests.tsx

```diff
@@ -324,10 +339,6 @@ class ZipInputTest extends React.Component<any, any> {
   }
 }
 
-///////////////////////////////
```

Comment:
We're not keeping this ordering so I figured it'd be better to just remove this as it's not become misleading